### PR TITLE
Feat(ui) ✨ : Add support for Avro editor

### LINKED ISSUE #9115: UI : Add support for Avro editor
Add support for Avro editor

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #9115 
### Describe your changes :

I worked on issue #9115, unit test is pending for the new component and I will add that in the next patch.

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/TopicDetails/TopicSchema/TopicSchema.tsx [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+        dataIndex: 'description',
+        key: 'description',
+        accessor: 'description',
+        render: renderFieldDescription,
+      },
+      {
+        title: t('label.tag-plural'),
+        dataIndex: 'tags',
+        key: 'tags',
+        accessor: 'tags',
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TopicDetails/TopicSchema/TopicSchema.tsx:275]: What is an accessor? why do we need this?

File: openmetadata-ui/src/main/resources/ui/src/components/TopicDetails/TopicSchema/TopicSchema.tsx [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+
+    expandIcon: ({ expanded, onExpand, expandable, record }) =>
+      expandable && (
+        <Typography.Text
+          className="m-r-xs cursor-pointer"
+          onClick={(e) => onExpand(record, e)}>
+          <FontAwesomeIcon icon={expanded ? faCaretDown : faCaretRight} />
+        </Typography.Text>
+      ),
+  };
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TopicDetails/TopicSchema/TopicSchema.tsx:303]: Let's move this to tableUtils mostly? so use it generic way?

**Review by chirag-madlani — CHANGES_REQUESTED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/TopicDetails/TopicSchema/TopicSchema.tsx (hunk 1 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/TopicDetails/TopicSchema/TopicSchema.tsx:275]: It's not needed, I will remove that.

File: openmetadata-ui/src/main/resources/ui/src/components/TopicDetails/TopicSchema/TopicSchema.tsx (hunk 2 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/TopicDetails/TopicSchema/TopicSchema.tsx:303]: Sure, I will update it.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**